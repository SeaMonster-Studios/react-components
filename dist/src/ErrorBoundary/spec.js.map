;;;;AAAA;;;;AACA;;AAEA;;AACA;;;;;;;;;AAFA;;;AAIA,IAAMA,cAAcC,QAAQC,GAAR,CAAYC,QAAhC;;AAEAC,UAAU,YAAM;AACdH,UAAQC,GAAR,CAAYC,QAAZ,GAAuB,YAAvB;AACD,CAFD;;AAIAE,SAAS,YAAM;AACbJ,UAAQC,GAAR,CAAYC,QAAZ,GAAuBH,WAAvB;AACD,CAFD;;AAIAM,UAAUC,4BAAV;;AAEAC,SAAS,8BAAT,EAAyC,YAAM;AAC7CC,KAAG,6CAAH,EAAkD,YAAM;AAAA,uBACpCC,aADoC;AAAA,QAC9CC,KAD8C,gBAC9CA,KAD8C;;AAGtDC,WAAOD,KAAP,EAAcE,WAAd;AACD,GAJD;;AAMAJ,KAAG,8DAAH,EAAmE,YAAM;AACvE;AACAK,YAAQC,KAAR,GAAgB;AAAA,aAAMC,SAAN;AAAA,KAAhB;;AAFuE,wBAI3BN,aAJ2B;AAAA,QAI/DO,SAJ+D,iBAI/DA,SAJ+D;AAAA,QAIpDC,aAJoD,iBAIpDA,aAJoD;AAAA,QAIrCP,KAJqC,iBAIrCA,KAJqC;;AAMvEC,WAAOD,KAAP,EAAcE,WAAd;;AAEA,QAAI;AACF,+BAAWF,KAAX;AACD,KAFD,CAEE,OAAOI,KAAP,EAAc;AACdI;AACD;;AAEDP,WAAOM,cAAc,0BAAd,CAAP,EAAkDL,WAAlD;AACAD,WAAOM,cAAc,+BAAd,CAAP,EAAuDL,WAAvD;AACAD,WAAOK,UAAUG,UAAjB,EAA6BC,eAA7B;AACD,GAjBD;AAkBD,CAzBD;;AA2BA,IAAMF,eAAe,SAAfA,YAAe;AAAA,SAAM,IAAN;AAAA,CAArB;;AAEA,SAAST,WAAT,GAAuB;AAAA,MACfY,KADe;AAAA;;AAAA;AAAA;;AAAA;;AAAA;;AAAA;AAAA;AAAA;;AAAA,sLAEnBC,KAFmB,GAEX;AACNC,iBAAS;AADH,OAFW;AAAA;;AAAA;AAAA;AAAA,+BAKV;AAAA;;AACP,eACE;AAAA;AAAA;AACE,2BAAY,qCADd;AAEE,qBAAS;AAAA,qBACP,OAAKC,QAAL,CAAc,UAACC,SAAD;AAAA,yCACTA,SADS;AAEZF,2BAAS;AAFG;AAAA,eAAd,CADO;AAAA;AAFX;AAUG,eAAKD,KAAL,CAAWC,OAAX,GAAqB,8BAAC,UAAD,OAArB,GAAsC;AAVzC,SADF;AAcD;AApBkB;;AAAA;AAAA,IACDG,gBAAMC,SADL;;AAuBrB,MAAMC,UAAU,iCACd;AAAC,mBAAD;AAAA;AACE,kCAAC,KAAD;AADF,GADc,CAAhB;AAKA,MAAMlB,QAAQkB,QAAQC,WAAR,CAAoB,qCAApB,CAAd;;AAEA,2BACKD,OADL;AAEElB;AAFF;AAID