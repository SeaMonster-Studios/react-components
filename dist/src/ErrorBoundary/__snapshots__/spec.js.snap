// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary Component Test Renders error message and report form when there is an error 1`] = `
<div
  class="css-975bms etl3wss0"
  data-testid="component-error-boundary"
>
  <div>
    <h2>
      Sorry, something went wrong.
    </h2>
    <p
      data-testid="component-error-boundary-form"
    >
      This error has been reported to our development team. Please
       
      <button>
        click here to fill out a report.
      </button>
    </p>
    <p>
      Please try reloading the page.
    </p>
  </div>
</div>
`;
